src/main/java/ua/utility/kfsdbupgrade/ForeignKeyReference.java

```diff
@@ -105,7 +105,13 @@ private String buildIndexName(int indx) {
 		StringBuilder retval = new StringBuilder(128);
 		retval.append(schemaName.toUpperCase());
 		retval.append(".");
-		retval.append(indexNameTemplate.replace("[table-name]", tableName).replace("{index}", "" + indx));
+		// Oracle before v 12.2 only allows identifiers upto 30 characters in length, so shorten index names
+		String shortTableName = tableName;
```

Comment:
Yes, not a bad solution for indexes, which aren't mapped. I'd suggest just changing the table name in both projects. Not as cool as being dynamic, but I think we should have a limited number of these cases.